### PR TITLE
Minor fix text align Translation for RTL languages

### PR DESCRIPTION
**Pull request by mehradi-github**

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32937

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32938

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32939

File: src/components/Translation/index.js [hunk 1]
```diff
@@ -40,15 +40,15 @@ class Translation extends Component {
...
+    const isRtl = (lang === 'arabic' || lang === 'persian' || lang === 'urdu' || lang === 'punjabi');
     return (
       <div
         id={`trans${index}`}
         className={`${styles.translation} ${isArabic && 'arabic'} translation`}
       >
         <h4 className="montserrat">{translation.resourceName}</h4>
         <h2
-          className={`${isArabic ? 'text-right' : 'text-left'} text-translation times-new`}
+          className={`${isRtl ? 'lang-dir-rtl' : 'text-left'} text-translation times-new`}
```
Comment (naveed-ahmad) [inline src/components/Translation/index.js]: I would say add language name as css class in parent div and control text alignment using css. 

```css
className={`${styles.translation} ${isArabic && 'arabic'} ${lang} translation`}
```
Using this approach, we can: 
- Configure language specific fonts ( like we've using nafees font for Urdu) 
- Control the alignment and maybe more in future  
- And this approach is DRY(we've tafisr and footnotes in multiple languages)  

We even don't need `text-left` or `text-right` classes here.

Comment (mehradi-github) [reply src/components/Translation/index.js]: Of course it's better

**Comment by mehradi-github** [comment]
Dear naveed-ahmad, Thank you for your attention

**Pull request closed without merging**